README.md

```diff
@@ -44,159 +40,158 @@ In addition, the file name should be of the format `time-report-x.csv`,
 where `x` is the ID of the time report. For example, `time-report-42.csv` would
 represent a report with an ID of `42`.
 
-Our partner has guaranteed that:
+In addition, the file name of the time report should be of the format `time-report-x.csv`, where `x` is the ID of the time report. For example, `time-report-42.csv` would represent a report with an ID of `42`.
```

Comment:
We should update the playbook or rubric to assess what the submission does when there are 0 or more rows given we specify 0 as a valid CSV here.